packages/create-cloudflare/src/frameworks/next/index.ts

```diff
@@ -22,34 +28,58 @@ const generate = async (ctx: PagesGeneratorContext) => {
 	);
 };
 
+const getApiTemplate = (
+	apiPath: string,
+	isTypescript: boolean
+): [string, string] => {
```

Comment:
I am not the biggest fan of returning tuples, sorry I am not too familiar with this codebase, is it common here?

Reply:
The previous version did it with a ternary as it assumed there was only one type of template that could be used, but since there are two types, I moved the logic deciding which path+template to use to its own function. When doing this though, I kept the same way that was used previously - using an array that gets destructured.

Reply:
but before it was within a function and with well named variables 🤔 

now it's a return value

as I said I am not the biggest fan (as I personally find it more difficult to read and more error-prone) but I am ok with it if others are 🙂